patches/server/0184-Fix-Citizens.patch

```diff
@@ -0,0 +1,106 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Bierque Jason <bierquejason@gmail.com>
+Date: Wed, 11 Jan 2023 23:10:20 +0100
+Subject: [PATCH] Fix Citizens
+
+
+diff --git a/src/main/java/net/minecraft/stats/ServerStatsCounter.java b/src/main/java/net/minecraft/stats/ServerStatsCounter.java
+index f62c4e97b294b697d5a68affa25f9904f8ea7cb7..11c2aad228d33c5691f21e8697ff3ae92c271d9e 100644
+--- a/src/main/java/net/minecraft/stats/ServerStatsCounter.java
++++ b/src/main/java/net/minecraft/stats/ServerStatsCounter.java
+@@ -55,8 +55,17 @@ public class ServerStatsCounter extends StatsCounter {
+         // MultiPaper start
+         if (true) {
+             try {
+-                String json = ExternalPlayer.loadedStats.remove(UUID.fromString(this.file.getName().split("\\.")[0]));
+-                if (json == null) json = MultiPaper.readStats(server.storageSource.levelDirectory.path().getFileName().toString(), this.file.getName().split("\\.")[0]);
++                // DoctaEnkoda - Start Fix Citizens
```

Comment:
You should use `MultiPaper`  instead of your name here

Reply:
Yes it will be done in the end, it's just that for the moment, I preferred to put my nickname to find myself in the code to see what I modified.

Reply:
> Yes it will be done in the end, it's just that for the moment, I preferred to put my nickname to find myself in the code to see what I modified.

here is a tip of better way
`MultiPaper - Ham1255 modified this code!`

Reply:
Indeed, it's a good idea!